packages/teams-js/test/public/sharing.spec.ts

```diff
@@ -23,8 +25,33 @@ describe('sharing_v1', () => {
     }
   });
 
+  Object.keys(FrameContexts)
+    .map(key => FrameContexts[key])
```

Comment:
Nit: I think this is fine as is, but for future tests you could use Object.values(FrameContexts) without doing the extra mapping.